tests/OpenToolkit.OpenCL.Tests/Program.cs

```diff
@@ -88,9 +107,9 @@ static void Main(string[] args)
 					});
 
 					//get rid of the buffers because we no longer need them
-					CL.ReleaseMemObject(bufferA);
-					CL.ReleaseMemObject(bufferB);
-					CL.ReleaseMemObject(resultBuffer);
+					CL.ReleaseMemoryObject(bufferA);
+					CL.ReleaseMemoryObject(bufferB);
+					CL.ReleaseMemoryObject(resultBuffer);
```

Comment:
We might want to name this `CL.ReleaseBuffer`.

Reply:
In OpenCL Buffers, Images and Pipes are all Memory Objects, I could rename the overloads but I am not sure if it's that a good idea